main.go

```diff
@@ -29,12 +29,16 @@ import (
 
 	clientset "github.com/xmudrii/etcdproxy-controller/pkg/client/clientset/versioned"
 	informers "github.com/xmudrii/etcdproxy-controller/pkg/client/informers/externalversions"
+	"github.com/xmudrii/etcdproxy-controller/pkg/controller/etcdproxy"
 	"github.com/xmudrii/etcdproxy-controller/pkg/signals"
 )
 
 var (
-	masterURL  string
 	kubeconfig string
+
+	etcdURL             string
```

Comment:
can we qualify all core etcd variable with "core"? We have too many etcds around.

Reply:
Done.